test/unit/gemm/device/gemm_testbed_3x.hpp

```diff
@@ -1359,6 +1360,12 @@ struct TestbedImpl {
     //
 
     size_t smem_size = static_cast<size_t>(Gemm::GemmKernel::SharedStorageSize);
+    size_t device_smem_size;
+#if defined(CUTLASS_ENABLE_SYCL)
+    syclcompat::device_info info = syclcompat::get_current_device().get_device_info();
+    this->sm_count = info.get_max_compute_units();
```

Comment:
The value returned by `get_max_compute_units()` is *not* what's likely to be expected on Intel hardware (448 instead of 56) as per our ongoing discussions. Not sure if we need to deal with that here.